packages/mobile/src/tokens/saga.ts

```diff
@@ -248,3 +253,23 @@ export async function getCurrencyAddress(currency: Currency) {
       return contractKit.registry.addressFor(CeloContract.StableTokenEUR)
   }
 }
+
+export function* importTokenInfo() {
+  const tokens: Token[] = yield readOnceFromFirebase('tokensInfo')
+  let token: Token
+  const kit: ContractKit = yield call(getContractKit)
+  const address: string = yield select(currentAccountSelector)
+  const balances: TokenBalances = {}
+  for (token of tokens) {
+    const contract = new kit.web3.eth.Contract(erc20.abi, token.address)
+    const balance: number = yield call(contract.methods.balanceOf(address).call)
```

Comment:
For the balance, I think it makes sense to store it in the readable format so that it can be used directly elsewhere without worrying about the decimals.

This means calling `new BigNumber(balance).dividedBy(new BigNumber(10).exponentiatedBy(token.decimals))`

Reply:
If you need a token to test this decimals thing you can use this test token: https://test-token-faucet.web.app/